week-2/E-conditionals/exercise.js

```diff
@@ -4,18 +4,19 @@
   Add an if statement to only show the name if the role is student
 */
 
-
-var name = "Daniel";
-var role = "mentor";
+var name1 = "Daniel";
+var role1 = "mentor";
 
 var name2 = "Harun";
 var role2 = "student";
 
+if (role1 === "student") {
+  console.log("Hi, My name is " + name);
+}
 
-console.log("Hi, My name is " + name);
-console.log("Hi, My name is " + name2);
-
-
+if (role2 === "student") {
+  console.log("Hi, My name is " + name2);
+}
```

Comment:
Could you write a function to handle the task, rather than repeating the same `if` statement with different names/roles?